lib/streamlit/elements/data_editor.py

```diff
@@ -88,13 +88,17 @@
         # TODO(lukasmasuch): Add support for np.ndarray
         # but it is not possible with np.ndarray.
         # NDArray[Any] works, but is only available in numpy>1.20.
+        # TODO(lukasmasuch): Typing for pa.Table is also not possible here because
+        # it would result in any since it is a C-based class.
+        # pa.Table,
```

Comment:
Is this comment missing something?

Reply:
this is only added since, theoretically, `pa.Table` should be listed here as type, but it's not working since `pa.Table` just results in Any.

Reply:
I rewrote the comment a bit